app/inputs/multi_value_input.rb

```diff
@@ -0,0 +1,64 @@
+class MultiValueInput < SimpleForm::Inputs::Base
+  def input(wrapper_options)
+    input_html_classes.unshift("string")
+    input_html_options[:name] ||= "#{object_name}[#{attribute_name}][]"
+
+    template.content_tag("ul", :class => "listing") do
+      collection_inputs
+    end
+  end
+
+  protected
+    def collection_inputs
+      tags = []
+      collection.each_with_index do |value, index|
+        tags << template.content_tag("li", :class => "field-wrapper") do
+          build_field(value, index)
+        end
+      end
+      tags.reduce(:<<)
+    end
+
+    def build_field_options(value, index)
+      options = input_html_options.dup
+
+      # Defaults and overrides
+      options[:value] = value
+      options[:id] ||= input_dom_id
+      options[:class] ||= []
+      options[:class] += ["#{input_dom_id} form-control multi-text-field"]
+      options[:'aria-labelledby'] = label_id
+
+      # ID must be unique, and required fields can't be infinitely required
+      if index > 0
+        options[:id] = nil
+        options[:required] = nil
+      end
+
+      options
+    end
+
+    def build_field(value, index)
+      options = build_field_options(value, index)
+      case options.delete(:type)
+        when 'textarea'.freeze
+          @builder.text_area(attribute_name, options)
+        when 'uri_enabled_string'.freeze
```

Comment:
Okay. Since we have to do this for textarea anyways. I'm imagining a future big case statement trying to take into account various types of fields that might be returned.